### ISSUE TITLE
[Bug][A11y][Suggestion Chip] Wrong label

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#519]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS26

### App version

0.6.0 (520)

### Describe the issue

Suggestion chip doesn't use the visible label as an accessibility label, the accessible label is always "Chip label", whatever the customized label is.

<img width="291" height="93" alt="" src="https://github.com/user-attachments/assets/b74ed319-e841-4b82-8e5d-2c4337dc6dbc" />

### Expected Behavior

Use only the visible label as an accessible label, remove the "chip" role from the label

**Comment by nouha06** [comment]
@pya35 waht about the suggestion chip with icon only, what we should hear from voice over ?

**Comment by pya35** [comment]
@nouha06 the label for chip with icon only should be customisable by the project. 
For the demo purpose, you can set a label related to the icon (for example "add to favourites" for the heart icon)

**Comment by pya35** [comment]
OK
